sources/packages/backend/libs/services/src/confirmation-of-enrollment/confirmation-of-enrollment.service.ts

```diff
@@ -437,6 +450,23 @@ export class ConfirmationOfEnrollmentService {
             `While updating COE status to '${COEStatus.declined}' the number of affected row was bigger than the expected one. Expected 1 received ${updateResult.affected}`,
           );
         }
+        this.logger.log(
+          "Assessing if there is a future impacted application that need to be reassessed.",
+        );
+        const impactedApplication =
+          await this.assessmentSequentialProcessingService.assessImpactedApplicationReassessmentNeeded(
+            studentAssessmentId,
+            this.systemUserService.systemUser.id,
+            transactionalEntityManager,
+          );
+
+        if (impactedApplication) {
+          this.logger.log(
+            `Application id ${impactedApplication.id} was detected as impacted and will be reassessed.`,
+          );
+        } else {
+          this.logger.log("No impacts were detected on future applications.");
```

Comment:
Do we need this else loop just for a log?

Reply:
Even the if condition here exclusively for logging. I idea here is to be more precise on what happened on assess impacted application.

Please let me know if there is a different suggested approach.